src/ray/gcs/client.h

```diff
@@ -0,0 +1,81 @@
+#ifndef RAY_GCS_CLIENT_H
+#define RAY_GCS_CLIENT_H
+
+#include <map>
+#include <string>
+
+#include "plasma/events.h"
+#include "ray/id.h"
+#include "ray/status.h"
+#include "ray/gcs/tables.h"
+#include "ray/util/logging.h"
```

Comment:
would be more alphabetical to move `ray/gcs` before `ray/id`

Reply:
things are ordered in such a way that includes that are in more nested directories are further down